src/Model/VideoManager.php

```diff
@@ -2,25 +2,24 @@
 
 namespace Mi\VideoManagerPro\SDK\Model;
 
+use JMS\Serializer\Annotation\Type;
+
 /**
  * @author Steve Reichenbach <steve.reichenbach@movingimage.com>
  *
  * @codeCoverageIgnore
  */
 class VideoManager
 {
+    /**
+     * @Type("string")
+     */
     private $id;
-    private $name;
 
     /**
-     * @param $name
-     * @param $id
+     * @Type("string")
      */
-    public function __construct($name, $id)
```

Comment:
Yes, I've checked. Only in some tests.
All phpunit tests are passed, but I didn't investigate why those tests are still working!